torch/export/exported_program.py

```diff
@@ -681,6 +681,28 @@ def _decompose_exported_program(
     return exported_program
 
 
+@contextmanager
```

Comment:
Ah, what I had mind with this was to wrap it around some broad chunk of code in export (maybe _export_func), so that anyone working on export who doesn't know about this issue can just call `state_dict()`. That way we also don't have to do the manual construction from named_parameters + named_buffers.